src/main/java/com/exasol/adapter/dialects/elasticsearch/ColumnNameMapper.java

```diff
@@ -0,0 +1,37 @@
+package com.exasol.adapter.dialects.elasticsearch;
+
+/**
+ * Map Exasol column names to ElasticSearchSQL column names and vice versa.
+ *
+ * NOTE: This is necessary because ElasticSearch nested column names are case sensitive dot-separated names, as in
+ * `book.author.name`. We cannot use this identifier in the Exasol database when performing queries, as the Exasol SQL
+ * dialect doesn't allow dots(.) in quoted identifiers.
+ */
+public class ColumnNameMapper {
```

Comment:
```suggestion
public final class ColumnNameMapper {
```
If you make a pure-static class, always make it final. This one is stateless and dead simple, so static is acceptable. In all other cases, create a proper instance.